### ISSUE TITLE
[BUG][WEBJS] - Internal server error when trying to send video using sendVideo endpoint.

### ISSUE THREAD

**Issue by arielpts** [allburov/whatsapp-http-api#629]
### Describe the bug
We are getting internal server error when trying to send video using sendVideo endpoint.
### Version
Get the WAHA version by calling `GET /api/version`
```json
{
  "version": "2024.11.2",
  "engine": "WEBJS",
  "tier": "PLUS",
  "browser": "/usr/bin/google-chrome-stable"
}
```
## Steps
**To Reproduce** Steps to reproduce the behavior:
1. Open swagger url: /#/:outbox_tray:%20Chatting/ChattingController_sendVideo
3. Change session and chatId, and send the example json.
### Expected behavior
Should send video to the given chatId.
### Requests - Responses
Request:
You can use example json.
Response:
```json
{
  "statusCode": 500,
  "timestamp": "2024-11-06T18:06:36.360Z",
  "exception": {
    "stack": "TargetCloseError: Protocol error (Runtime.callFunctionOn): Target closed\n    at CallbackRegistry.clear (/app/node_modules/puppeteer-core/lib/cjs/puppeteer/common/CallbackRegistry.js:73:36)\n    at CdpCDPSession._onClosed (/app/node_modules/puppeteer-core/lib/cjs/puppeteer/cdp/CDPSession.js:101:25)\n    at #onClose (/app/node_modules/puppeteer-core/lib/cjs/puppeteer/cdp/Connection.js:167:21)\n    at WebSocket.<anonymous> (/app/node_modules/puppeteer-core/lib/cjs/puppeteer/node/NodeWebSocketTransport.js:49:30)\n    at callListener (/app/node_modules/ws/lib/event-target.js:290:14)\n    at WebSocket.onClose (/app/node_modules/ws/lib/event-target.js:220:9)\n    at WebSocket.emit (node:events:520:28)\n    at WebSocket.emitClose (/app/node_modules/ws/lib/websocket.js:272:10)\n    at Socket.socketOnClose (/app/node_modules/ws/lib/websocket.js:1341:15)\n    at Socket.emit (node:events:520:28)\n    at TCP.<anonymous> (node:net:338:12)\n    at TCP.callbackTrampoline (node:internal/async_hooks:130:17)",
    "message": "Protocol error (Runtime.callFunctionOn): Target closed",
    "name": "TargetCloseError",
    "cause": {
      "stack": "ProtocolError\n    at <instance_members_initializer> (/app/node_modules/puppeteer-core/lib/cjs/puppeteer/common/CallbackRegistry.js:94:14)\n    at new Callback (/app/node_modules/puppeteer-core/lib/cjs/puppeteer/common/CallbackRegistry.js:98:16)\n    at CallbackRegistry.create (/app/node_modules/puppeteer-core/lib/cjs/puppeteer/common/CallbackRegistry.js:23:26)\n    at Connection._rawSend (/app/node_modules/puppeteer-core/lib/cjs/puppeteer/cdp/Connection.js:90:26)\n    at CdpCDPSession.send (/app/node_modules/puppeteer-core/lib/cjs/puppeteer/cdp/CDPSession.js:66:33)\n    at #evaluate (/app/node_modules/puppeteer-core/lib/cjs/puppeteer/cdp/ExecutionContext.js:360:50)\n    at ExecutionContext.evaluate (/app/node_modules/puppeteer-core/lib/cjs/puppeteer/cdp/ExecutionContext.js:275:36)\n    at IsolatedWorld.evaluate (/app/node_modules/puppeteer-core/lib/cjs/puppeteer/cdp/IsolatedWorld.js:100:30)\n    at CdpFrame.evaluate (/app/node_modules/puppeteer-core/lib/cjs/puppeteer/api/Frame.js:341:43)\n    at CdpFrame.<anonymous> (/app/node_modules/puppeteer-core/lib/cjs/puppeteer/util/decorators.js:104:27)\n    at CdpPage.evaluate (/app/node_modules/puppeteer-core/lib/cjs/puppeteer/api/Page.js:807:43)\n    at WebjsClient.sendMessage (/app/node_modules/whatsapp-web.js/src/Client.js:956:44)\n    at WhatsappSessionWebJSPlus.sendVideo (/app/dist/plus/engines/webjs/session.webjs.plus.js:56:30)\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)",
      "message": "",
      "name": "ProtocolError"
    }
  },
  "request": {
    "path": "/api/sendVideo",
    "method": "POST",
    "body": {
      "chatId": "...@c.us",
      "file": {
        "mimetype": "video/mp4",
        "filename": "video.mp4",
        "url": "https://github.com/devlikeapro/waha/raw/core/examples/video.mp4"
      },
      "reply_to": null,
      "asNote": false,
      "caption": "Just watch at this!",
      "session": "dev"
    },
    "query": {}
  },
  "version": {
    "version": "2024.11.2",
    "engine": "WEBJS",
    "tier": "PLUS",
    "browser": "/usr/bin/google-chrome-stable"
  }
}
```

[![patron:PRO](https://img.shields.io/badge/patron-PRO-188a42)](https://waha.devlike.pro/docs/how-to/plus-version/#tiers)

**Comment by devlikepro** [comment]
working on the fix

[![patron:PRO](https://img.shields.io/badge/patron-PRO-188a42)](https://waha.devlike.pro/docs/how-to/plus-version/#tiers)

**Comment by devlikepro** [comment]
Fix in `2024.11.3`!

[![patron:PRO](https://img.shields.io/badge/patron-PRO-188a42)](https://waha.devlike.pro/docs/how-to/plus-version/#tiers)
